services/bots/src/github-webhook/github-webhook.service.ts

```diff
@@ -13,8 +13,33 @@ export class GithubWebhookService {
   }
 
   async handleWebhook(params: WebhookHandlerParams): Promise<void> {
+    ISSUE_UPDATES[params.deliveryId] = {
```

Comment:
All code that handles the mutation of this global variable should be in 1 file.

Let's also create helper functions to get/set/clean it, and instead of deliveryId, make it `getContext(uniqueId)` etc. So it could be reused for something else.